app/src/backend/formulas.ts

```diff
@@ -216,9 +216,10 @@ export function handleIndustrialWasteWaterFormula(
   const degradableOrganicComponents = data["degradable-organic-components"];
   const methaneProductionCapacity =
     data["methane-production-capacity"] ?? DEFAULT_METHANE_PRODUCTION_CAPACITY; // TODO should this only be handled UI-side?
-  const removedSludge = data["removed-sludge"];
-  const methaneCorrectionFactor = data["methane-correction-factor"];
-  const methaneRecovered = data["methane-recovered"];
+  const removedSludge = data["total-organic-sludge-removed"];
+  const methaneCorrectionFactor = data["methane-correction-factor"]; // TODO @leminkoh is this correct? it doesn't exist on the ui
```

Comment:
I don't if this is correct @lemilonkh  because this field doesn't exist on the UI

Reply:
This needs to come from the FormulaInput table (it's called MCF there
). Alternatively for now you can encode similar if/ else rules like we have for other formulas to not have to load this data until it's ready.

Reply:
I'll extend this in a follow up ticket

Reply:
As long as it doesn't give us an error with saving for now that's good. Otherwise comment this line and replace it with a constant for now.

Reply:
Because the result will be NaN otherwise